featurehub_sdk/version.py

```diff
@@ -1 +1 @@
-sdk_version="0.0.2"
+sdk_version="1.0.3"
```

Comment:
you went from 0.0.2 to 1.0.3  - doesn't look right?

Reply:
It should never have been released as 0.X, that doesn't instill confidence. Perhaps when you release it, make it 0.0.3 and then bump the version to 1.0.0 and release it again?

Reply:
why don't you change it to 1.0.0 if you want to make a major release? This however usually means breaking changes